File: key-journal-front/src/app/models/UserEDP.ts
export class UserEDP {

    id?: number;
    acckId: number;
    acckName?: string;
    acck: Map<number, string>;
    userPIB: string;
    userResponsiblePIB: string;
    dateFrom: Date;
    dateTo: Date;
    identityZNOK: string;

    
    constructor(id: number, acckId: number, acckName: string, userPIB: string,
        userResponsiblePIB: string, dateFrom: Date, dateTo: Date, identityZNOK: string, acck: Map<number, string>) {
        this.acckId = acckId;
        this.acckName = acckName;
        this.userPIB = userPIB;
        this.userResponsiblePIB = userResponsiblePIB;
        this.dateFrom = dateFrom;
        this.dateTo = dateTo;
        this.identityZNOK = identityZNOK;
        this.acck = acck;
    }
}

File: kep-journal-backend/src/main/java/com/dkey/kep/model/UserEDP.java
package com.dkey.kep.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;
import java.time.LocalDate;

@Entity
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
public class UserEDP {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
    @ManyToOne(optional = false)
    private ACCK acck;

    private String userPIB;
    private String userResponsiblePIB;
    private LocalDate dateFrom;
    private LocalDate dateTo;
    private String identityZNOK;

}
